docs/source/using/views.rst

```diff
@@ -1,20 +1,44 @@
+Starting the Workflow Manager
+=============================
+
+The Workflow Manager can be started from within the ``edm`` environment with,
+
+.. code-block:: console
+
+    (edm)$ edm run -e force-pyXX -- force_wfmanager
+
+Alternatively one can enter the ``force-pyXX`` environment,
+
+.. code-block:: console
+
+    (edm)$ edm shell -e force-pyXX
+
+and then,
+
+.. code-block:: console
+
+    (force-pyXX)(edm)$ force_wfmanager
```

Comment:
Note: This section might needed to be updated, depending on what syntax we decide in the other page